3-appfactory/modules/app-group-baseline/outputs.tf

```diff
@@ -14,3 +14,47 @@
  * limitations under the License.
  */
 
+output "app_admin_project_id" {
+  description = "Project ID of the application admin project."
+  value       = module.app_admin_project.project_id
+}
+
+output "app_infra_repository_name" {
+  description = "Name of the application infrastructure repository."
+  value       = google_sourcerepo_repository.app_infra_repo.name
+}
+
+output "app_infra_repository_url" {
+  description = "URL of the application infrastructure repository."
+  value       = google_sourcerepo_repository.app_infra_repo.url
+}
+
+output "app_cloudbuild_workspace_apply_trigger_id" {
+  description = "ID of the apply cloud build trigger."
+  value       = module.tf_cloudbuild_workspace.cloudbuild_apply_trigger_id
+}
+
+output "app_cloudbuild_workspace_plan_trigger_id" {
+  description = "ID of the plan cloud build trigger."
+  value       = module.tf_cloudbuild_workspace.cloudbuild_plan_trigger_id
+}
+
+output "app_cloudbuild_workspace_artifacts_bucket_name" {
+  description = "Artifacts bucket name for the application workspace."
+  value       = module.tf_cloudbuild_workspace.artifacts_bucket
+}
+
+output "app_cloudbuild_workspace_logs_bucket_name" {
+  description = "Logs bucket name for the application workspace."
+  value       = module.tf_cloudbuild_workspace.logs_bucket
+}
+
+output "app_cloudbuild_workspace_state_bucket_name" {
+  description = "Terraform state bucket name for the application workspace."
+  value       = module.tf_cloudbuild_workspace.state_bucket
+}
+
+output "app_env_project_ids" {
+  description = "Application environment projects IDs"
```

Comment:
`Application environment projects IDs`

Reply:
@renato-rudnicki could you add a suggestion for this case? 
I was not able to find the difference, is it the missing full stop?